### ISSUE TITLE
Implement --plain-http as a configuration 

### ISSUE THREAD

**Issue by SteveLasker** [notaryproject/notation#85]
For local testing and IoT scenarios that use plain HTTP, we currently need to add `--plain-http` as a parameter.

- `notation sign --plain-http $IMAGE`
- `notation pull --plain-http --peek $IMAGE`
- `notation verify --plain-http $IMAGE`

Alternatively, what if we added insecureRegistries as a configuration option:

```json
"insecureRegistries": [
    "registry.wabbit-networks.io"
  ]
```
## Expected Behavior

- `notation sign $IMAGE`
- `notation --list $IMAGE`
- `notation verify $IMAGE`

@shizhMSFT

**Comment by shizhMSFT** [comment]
Will be included in the next iteration.

**Comment by shizhMSFT** [comment]
Resolved by 07dada7

**Comment by SteveLasker** [comment]
I tried the latest [0.5.2 release](https://github.com/shizhMSFT/notation/releases/tag/v0.5.2), but I wasn't able to remove the `--plain-http` parameter.
I realized I didn't show the expected behavior in the initial issue, so I updated it.

**Comment by SteveLasker** [comment]
Resolved with the [0.5.2](https://github.com/shizhMSFT/notation/releases/tag/v0.5.2) release
